docs/reference/notices.md

```diff
@@ -1,13 +1,42 @@
 ---
 id: notices
 title: "Security notices"
-description: "Let's take a closer look at security notices, reporting vulnerabilities, and addiitonal security information."
+description: "Let's take a closer look at security notices, reporting vulnerabilities, and additional security information."
 ---
 
 ## Security notices
 
 Camunda publishes security notices after fixes are available.
 
+### Notice 14
+
+#### Publication date
+
+March 11th, 2025
+
+#### Product affected
+
+Camunda Web Modeler
+
+#### Impact
+
+The version of `koa` used by Camunda Web Modeler was affected by the following vulnerability:
+
+- https://nvd.nist.gov/vuln/detail/CVE-2025-25200
+
+#### How to determine if the installation is affected
+
+You are using Camunda Web Modeler version ≤ 8.3.16, ≤ 8.4.14, ≤ 8.5.15, or ≤ 8.6.7.
+
+#### Solution
+
+Camunda has provided the following releases which contain the fix:
+
+- Camunda Web Modeler 8.3.17
+- Camunda Web Modeler 8.4.15
+- Camunda Web Modeler 8.5.16
+- Camunda Web Modeler 8.6.8
```

Comment:
This should be backported so that when a user looks at their version of the docs, they can see there is a security notice. Tech writers can handle this for you today, so it's ready for tomorrow.